components/Home/Home.vue

```diff
@@ -50,7 +50,7 @@
           mode="out-in"
           :class="[
             'tw-hidden md:tw-block',
-            'flex-none tw-max-w-md tw-overflow-scroll',
+            'flex-none tw-max-w-[460px] tw-overflow-y-auto',
```

Comment:
Are we really require to hard code a size here?

Reply:
Yes because the previous value 'tw-max-w-md' (448px) didn't gave the component the width that it requires so I had to give it a bigger value (460px).